pinc/base.inc

```diff
@@ -23,6 +23,10 @@ if (!headers_sent()) {
     // Disallow other sites from embedding pages in frames/iframes
     header("Content-Security-Policy: frame-ancestors 'self'", false);
     header("X-Frame-Options: SAMEORIGIN");
+    header("X-Content-Type-Options: nosniff");
```

Comment:
This appears to enable [CORB](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/services/network/cross_origin_read_blocking_explainer.md) which may, or may not, be problematic for pgdp.net or other sites. We should do a bit of investigation and testing to confirm this won't break anything.

Reply:
I'll do that one in a separate MR later once I get some testing on it.